recipes/rb-data_uri/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "data_uri" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256:  7eb2f63487ccb943fae0eca561729c48d4d5654d76f8330aa16ed1dcdbebf33b
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+
+test:
+  commands:
+    - ruby -r {{ name }} -e 'exit 0'
+
+about:
+  home: https://rubygems.org/gems/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: {{ name }}-{{ version }}/README.rdoc
+  summary: Data URIs allow resources to be embedded inside a URI.
+  doc_url: https://www.rubydoc.info/gems/{{ name }}
+  dev_url: https://github.com/dball/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - kozo2
+    - sodre
```

Comment:
@sodre are you OK with being a maintainer?

Reply:
@scopatz, it is fine with me for the all ruby recipes until we set up a conda-forge/ruby group like they did for R.